spec/Section 6 -- Execution.md

```diff
@@ -396,10 +396,11 @@ entry from the grouped field set in the order provided in the grouped field set.
 It must determine the corresponding entry in the result map for each item to
 completion before it continues on to the next item in the grouped field set:
 
-For example, given the following selection set to be executed serially:
+For example, given the following mutation operation, the root _selection set_
+must be executed serially:
 
 ```graphql example
-{
+mutation {
```

Comment:
:new: Clarification

Reply:
define variable

Reply:
Perhaps we add the mutation example, and keep the selection set example?